tests/security/luks1/unlock_via_ssh_client.pm

```diff
@@ -33,20 +33,23 @@ sub run {
 
     # ssh to the server and unlock the boot partiton
     enter_cmd('ssh -i /root/.ssh/id_rsa root@10.0.2.101');
+    wait_still_screen(1);
     save_screenshot;
 
-    # We need add some sleep here to make sure each command can get return
-    sleep 5;
+    # We need wait a few seconds here to make sure each command can get return
+    wait_still_screen(5);
     send_key 'up';
     save_screenshot;
-    sleep 5;
+    wait_still_screen(5);
     send_key 'ret';
-    sleep 5;
+    wait_still_screen(5);
     save_screenshot;
     enter_cmd("$testapi::password");
-    sleep 5;
-    enter_cmd('exit');
-    sleep 5;
+    wait_still_screen(5);
+    save_screenshot;
+    # exit from ssh session if it is not closed by remote host
+    enter_cmd('exit') if !check_screen('ssh_connection_closed_by_remote_host');
```

Comment:
how about "unless"
anyway, not a big deal

Reply:
Done, thanks.